Gemfile

```diff
@@ -1,4 +1,6 @@
 source 'https://rubygems.org'
 
+gem 'slop', '~> 4.0', platforms: [:ruby_20, :ruby_21, :ruby_22]
```

Comment:
Right, it will work if installed through bundler but not with `gem install`. I'd just migrate to optparse then...